style.css

```diff
@@ -205,6 +213,61 @@ body {
   font-weight: 500;
   margin-bottom: 40px;
   border: none;
+  transition: all 1s linear;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+}
+
+.contact h2 {
+  color: #9390ff;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  width: 75%;
+}
+
+form input {
+  margin: 10px 0;
+  width: 100%;
+  height: 60px;
```

Comment:
- [ ] Please give horizontal padding for the input elements so that a text input appears a bit away from the borders of the input elements. 
- [ ] [OPTIONAL] Kindly decrease the horizontal margin of the form elements from the edge of the viewport. It is narrower in the design. 

<p align="center">&nbsp;
 
| Figma Design |   Your Project |
| ---------- | ----------- |
 | ![Template 4-main-mobile(1)](https://user-images.githubusercontent.com/12524453/177411508-d22e26ef-8f90-4979-8acc-c3267fb320dc.png) | ![Screen Shot 2022-07-05 at 23 31 24](https://user-images.githubusercontent.com/12524453/177411774-e4fd4ff2-31cb-4da5-ae8c-4812c375e71f.png) |
</p>